Gems/PhysX/Code/Source/EditorPrismaticJointComponent.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzFramework/Entity/EntityDebugDisplayBus.h>
+#include <AzToolsFramework/API/ComponentEntitySelectionBus.h>
+
+#include <PhysX/EditorJointBus.h>
+#include <Editor/EditorJointConfiguration.h>
+#include <Source/EditorJointComponent.h>
+
+namespace PhysX
+{
+    class EditorPrismaticJointComponent
+        : public EditorJointComponent
+    {
+    public:
+        AZ_EDITOR_COMPONENT(EditorPrismaticJointComponent, "{607B246E-C2DB-4D43-ABFA-A5A3994867C5}", EditorJointComponent);
+        static void Reflect(AZ::ReflectContext* context);
+
+        static void GetProvidedServices(AZ::ComponentDescriptor::DependencyArrayType& provided);
+        static void GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required);
+        static void GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible);
+
+        // AZ::Component
+        void Activate() override;
+        void Deactivate() override;
+
+        // EditorComponentBase
```

Comment:
uber nit: here and elsewhere in this file we're missing `overrides ...`

```
// AZ::Component overrides ... 
```

etc...

Reply:
Updated

Reply:
Thank you! 🙂